src/Model/QuestionManager.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace App\Model;
+
+class QuestionManager extends AbstractManager
+{
+    
+    const TABLE = 'question';
+
+    /**
+     *  Initializes this class.
+     */
+    public function __construct()
+    {
+        parent::__construct(self::TABLE);
+    }
+
+    /**
+     * @return array
+     * Selects a random question.
+     */
+    public function selectOneRandom(): array
+    {
+        $statement = $this->pdo->query("SELECT * FROM $this->table ORDER BY RAND() LIMIT 1");
+
+        return $statement->fetch();
```

Comment:
PDOStatement::fetch() could also return false, in case of no record in the database for instance.
If it happends, you'll have a PHP error because your return type is array.
You probably should deal with the error.